tests/unit/issuing.test.js

```diff
@@ -118,6 +118,13 @@ describe('Verifiable Credential Issuing', () => {
     expect(result.results[0].verified).toBe(true);
   }, 30000);
 
+  test('Issuing should return an object with a proof, and it must pass validation (using proofValue).', async () => {
```

Comment:
do you need a test for the `.jws` value too?

Reply:
the tests cover that already (see changes from proofValue -> jws)